src/Microsoft.ML.Data/TrainCatalog.cs

```diff
@@ -635,7 +635,8 @@ internal RankingTrainers(RankingCatalog catalog)
     }
 
     /// <summary>
-    /// The central catalog for anomaly detection tasks and trainers.
+    /// Class used to create instances of anomaly detection components,
```

Comment:
>Class [](start = 8, length = 5)

Suggestion: we somehow need to convey to the user that this class is not directly used by the user, and they don't create it directly. Just a suggestion.

Reply:
That's a good idea. I could reference MLContext

Reply:
Class used **by an MLContext object** to create ...?